src/cache.js

```diff
@@ -63,7 +47,7 @@ class Cache {
       const resp = await got(url);
       const parsedBody = parseSchema(resp.body, url);
       if (this.ttl > 0) {
-        this.cache.setKey(url, { timestamp: Date.now(), body: parsedBody });
+        this.cache.setKey(url, { body: parsedBody });
```

Comment:
In principle, I could just directly store `parsedBody` here i.e:

```js
this.cache.setKey(url, parsedBody);
```

I think I am going to leave this as-is to leave open the possibility that I could store other things in the cached object here without making a breaking change.